### DISCUSSION TITLE
Unable to use Copy tasks and Move multiple tasks at once

### DISCUSSION THREAD

**Discussion by annie8910** [ivan-lednev/obsidian-day-planner#463] [Q&A]
Hello I am using day planner but then some problems occurred.
The computer system I am using is Windows 10. I want to use Copy tasks and Move multiple tasks at once, but it doesn't work no matter which button I press.
Is there any solution I can do now?

**Comment by ivan-lednev** [comment]
Hello, @annie8910,

Closing this as a duplicate of #462. It explains why this behavior has changed. You can also add a comment about a more usable implementation there.
